lib/paper_trail/reifier.rb

```diff
@@ -91,14 +91,17 @@ def init_unversioned_attrs(attrs, model)
       #
       # @api private
       def reify_attribute(k, v, model, version)
-        if model.has_attribute?(k)
-          model[k.to_sym] = v
-        elsif model.respond_to?("#{k}=")
-          model.send("#{k}=", v)
-        elsif version.logger
-          version.logger.warn(
-            "Attribute #{k} does not exist on #{version.item_type} (Version id: #{version.id})."
-          )
+        begin
+          if model.has_attribute?(k)
+            model[k.to_sym] = v
+          elsif model.respond_to?("#{k}=")
+              model.send("#{k}=", v)
```

Comment:
https://github.com/paper-trail-gem/paper_trail/issues/1467 notes that this is the line that can raise the error. If that is the case, should the begin/rescue wrap just this line?

Reply:
Thanks. I've double checked our Datadog logs and have confirmed that the bug report is correct.